src/test/java/io/vertx/kafka/client/common/tracing/TracingTest.java

```diff
@@ -298,7 +298,7 @@ private TestTracer(TestContext ctx) {
     }
 
     void init(String topic, int sent, int received) {
-      init(topic, sent, received, "localhost:9092", "localhost", "9092");
+      init(topic, sent, received, "localhost:9092,localhost:9093", "localhost", "9093");
```

Comment:
why different port here?

Reply:
I added a broker to be able test assignments. This test was failing with 9092.